src/main/java/mate/academy/webapp/controller/OrderController.java

```diff
@@ -47,7 +47,7 @@ public OrderResponseDto createOrder(Authentication authentication,
     public List<OrderResponseDto> getOrdersHistory(Authentication authentication,
                                                    Pageable pageable) {
         User user = (User) authentication.getPrincipal();
-        return orderService.getAll(user, pageable);
+        return orderService.getAll(user.getId(), pageable);
     }
```

Comment:
You mustn't push changes that don't involved by task

Reply:
Agree.